From: Paul Zander <negril.nx+gentoo@gmail.com>

Use tar.gz instead of zip to make CI happy.

diff --git a/modules/gapi/cmake/DownloadADE.cmake b/modules/gapi/cmake/DownloadADE.cmake
index e22c4f1..fb0c10a 100644
--- a/modules/gapi/cmake/DownloadADE.cmake
+++ b/modules/gapi/cmake/DownloadADE.cmake
@@ -1,7 +1,7 @@
 set(ade_src_dir "${OpenCV_BINARY_DIR}/3rdparty/ade")
-set(ade_filename "v0.1.2d.zip")
+set(ade_filename "v0.1.2d.tar.gz")
 set(ade_subdir "ade-0.1.2d")
-set(ade_md5 "dbb095a8bf3008e91edbbf45d8d34885")
+set(ade_md5 "a0fc51a1090100023316c08322e8b093")
 ocv_download(FILENAME ${ade_filename}
              HASH ${ade_md5}
              URL
